sucury.py

```diff
@@ -279,71 +266,123 @@ def draw_grid():
             rect = pygame.Rect(x, y, grid_size, grid_size)
             pygame.draw.rect(arena, GRID_COLOR, rect, 1)
 
-score = BIG_FONT.render("1", True, MESSAGE_COLOR)
-score_rect = score.get_rect(center=(WIDTH/2, HEIGHT/20+HEIGHT/30))
+###
+### Init parser to get args from CLI
+###
+def init_cli_parser():
+    parser = argparse.ArgumentParser(description="The classic snake game")
 
-main_menu()
+    parser.add_argument('--width', default=WIDTH,
+                        help="set window's width",
+                        type=int)
+    parser.add_argument('--height', default=HEIGHT,
+                        help="set window's height",
+                        type=int)
+    parser.add_argument('-g', '--grid-size', default=grid_size,
+                        help="set window's grid size",
+                        type=int)
 
-snake = Snake()    # The snake
-apple = Apple()    # An apple
+    return parser
 
-##
-## Main loop
-##
+###
+### Set args gotten from CLI 
+###
+def set_cli_args(args):
+    global WIDTH
+    global HEIGHT
+    global grid_size
+    
+    WIDTH = args['width']
+    HEIGHT = args['height']
+    grid_size = max(min(args['grid_size'], 100), 10)
 
-while True:
+if __name__ == "__main__":
+    parser = init_cli_parser()
+    args = vars(parser.parse_args())
+    set_cli_args(args)
 
-    for event in pygame.event.get():           # Wait for events
+    ##
+    ## Game implementation.
+    ##
 
-       # App terminated
-        if event.type == pygame.QUIT:
-            pygame.quit()
-            sys.exit()
+    pygame.init()
 
-        # Key pressed
-        if event.type == pygame.KEYDOWN:
-            if event.key == pygame.K_DOWN or event.key == pygame.K_s:    # Down arrow or S:  move down
-                snake.change_direction(DOWN)
-            elif event.key == pygame.K_UP or event.key == pygame.K_w:    # Up arrow or W:    move up
-                snake.change_direction(UP)
-            elif event.key == pygame.K_RIGHT or event.key == pygame.K_d: # Right arrow or D: move right
-                snake.change_direction(RIGHT)
-            elif event.key == pygame.K_LEFT or event.key == pygame.K_a:  # Left arrow or A:  move left
-                snake.change_direction(LEFT)
-            elif event.key == pygame.K_q:     # Q         : quit game
+    clock = pygame.time.Clock()
+
+    arena = pygame.display.set_mode((WIDTH, HEIGHT))
+
```

Comment:
Maybe it's a good idea to standardize formatting with a linter so we can get consistent whitespaces